README.md

```diff
@@ -67,4 +67,10 @@ We decided against the "have libmongocrypt do everything" approach because it co
 The first version of FLE is to get signal. If FLE becomes popular, further improvements will be made (removing mongocryptd process, support for more queries, better performance). libmongocrypt takes the same approach. Making it blazing fast and completely future-proof is not a high priority.
 
 ### Releasing ###
-To release create a new git tag, and update `MONGOCRYPT_VERSION` in mongocrypt.h.
\ No newline at end of file
+Do the following when releasing:
+- Update `MONGOCRYPT_VERSION` in mongocrypt.h.
+- Create a new git tag, like `1.0.0-beta123`.
```

Comment:
The order of this one and the next should be flipped.  The version with the .kts file at 1.0.0-beta123 has to also be tagged with 1.0.0-beta123